woocommerce-services.php

```diff
@@ -249,6 +256,18 @@ class WC_Connect_Loader {
 
 		public static function plugin_deactivation() {
 			wp_clear_scheduled_hook( 'wc_connect_fetch_service_schemas' );
+
+			/**
```

Comment:
Super nitpicky and no need to change but since `/**` are PHPDoc DocBlocks and [not everything can be documented with a DocBlock](https://docs.phpdoc.org/3.0/guide/getting-started/what-is-a-docblock.html#what-can-be-documented-in-your-source-code), this could be a `/*` comment. :shrug: Doesn't really hurt though.